### ISSUE TITLE
RFE: Better handling of .gitignore if it already exists on initial export

### ISSUE THREAD

**Issue by jhgarrison** [joyfullservice/msaccess-vcs-addin#575]
On initial export into a directory that is detected to be a git repo, the export creates a .gitignore mentioning all the things that shouldn't be committed.  However, if a .gitignore already exists, it is not updated with msaccess-vcs entries.  

It would be nice if, in this case, there was a warning and the export wrote to an alternate filename, for example, `msaccess-vcs.gitignore` so the user could merge this with the existing file.

**Comment by jhgarrison** [comment]
I fully agree that attempting to merge your changes with existing .gitignore would be impossible to do "right" for all use cases.  I guess what I'd like to see is to get a warning along the lines of:

```
.gitignore already exists, no changes made
To see what entries are recommended for msaccess-vcs repositories, see 
[link to github page showing recommended entries]
```

**Comment by mwolfe02** [comment]
@joyfullservice If I understand correctly, the real challenge is in trying to identify whether any given Export to Source is the **initial** export to source, correct?  

If the destination folder is part of a git repo--and that repo has no `.gitignore` file--then it's safe to assume this is likely the initial export and that you can safely create a template `.gitignore` file.  However, if a `.gitignore` file already exists, it's difficult to know whether this is the initial export of the .accdb (vs. the 50th export) or whether the repo has other non-Access resources in it.  That's why simple solutions like creating an `msaccess-vcs.gitignore` file won't necessarily work.  Unless users committed that file to the repo, it would keep getting recreated on each export.

Could those concerns be addressed by comments at the top of the `msaccess-vcs.gitignore` explaining the purpose of the file and also including a line to ignore that file itself?  For example:

```
##################################################################
## This file is a recommended .gitignore template for working with msaccess-vcs-addin
## In a repo with no .gitignore file, it gets created as `.gitignore`.  
## In repos with an existing .gitignore file, it gets created as `msaccess-vcs.gitignore`.
## If you have an existing .gitignore file, you may want to merge some or all of the lines
##      below into that file.
##################################################################

# Ignore this template file
msaccess-vcs.gitignore
```

Or are there other issues I'm not considering?
